apps/frontend/src/pages/quiz/quiz.tsx

```diff
@@ -35,22 +35,22 @@ const Quiz: React.FC = () => {
 				return <Analyzing onNext={handleNextStep} />;
 			}
 
-			case Step.BALANCE_WHEEL: {
-				return <BalanceWheel />;
-			}
-
 			case Step.INTRODUCTION: {
 				return <Introduction onNext={handleNextStep} />;
 			}
 
+			case Step.QUIZ: {
```

Comment:
Try to avoid the changes, that are not required in your PR

Reply:
Somehow these changes are already on main